pkg/cluster/internal/delete/delete.go

```diff
@@ -33,16 +33,17 @@ func Cluster(logger log.Logger, p providers.Provider, name, explicitKubeconfigPa
 		return errors.Wrap(err, "error listing nodes")
 	}
 
+	logger.V(0).Infof("Removing context %q from kubeconfig ...", name)
 	kerr := kubeconfig.Remove(name, explicitKubeconfigPath)
 	if kerr != nil {
 		logger.Errorf("failed to update kubeconfig: %v", kerr)
+		return err
 	}
 
+	logger.V(0).Infof("Deleting nodes for cluster %q ...", name)
```

Comment:
This doesn't tell the user if we actually found any containers and removed them.